docs/pages-for-subheaders/installation-requirements.md

```diff
@@ -55,68 +55,125 @@ If you are installing Rancher on a K3s cluster with **Raspbian Buster**, follow
 
+(*): Large deployments require [additional tuning and following of best practices](../reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md).
 
-### RKE2 Kubernetes
+(†): External Database Host refers to the optional possibility of hosting [k3s cluster data store on an external dedicated host](https://docs.k3s.io/datastore). Exact requirements will depend on the chosen data store, this table is a guideline only.
+
+Refer to k3s documentation for more detailed information on [k3s general requirements](https://docs.k3s.io/installation/requirements).
+
+### Hosted Kubernetes
+
+Minimum CPU and memory requirements for each individual node in the [Kubernetes cluster Rancher is installed in](install-upgrade-on-a-kubernetes-cluster.md) are listed in the table below.
 
-These CPU and memory requirements apply to each instance with RKE2 installed. Minimum recommendations are outlined here.
+Please note that a highly available setup with at least 3 nodes is required for all production usages.
 
-| Deployment Size | Clusters | Nodes     | vCPUs | RAM  |
-| --------------- | -------- | --------- | ----- | ---- |
-| Small           | Up to 5  | Up to 50  | 2     | 5 GB |
-| Medium          | Up to 15 | Up to 200 | 3     | 9 GB |
+These requirements apply hosted Kubernetes clusters such as EKS, AKS, or GKE. They do not apply to Rancher SaaS solutions such as Rancher Prime Hosted](https://www.rancher.com/products/rancher).
+
+| Managed Infrastructure Size | Clusters  | Nodes      | vCPUs | RAM   |
+|-----------------------------|-----------|------------|-------|-------|
+| Small                       | Up to 150 | Up to 1500 | 4     | 16 GB |
+| Medium                      | Up to 300 | Up to 3000 | 8     | 32 GB |
+| Large (*)                   | Up to 500 | Up to 5000 | 16    | 64 GB |
+
+(*): Large deployments require [additional tuning and following of best practices](../reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md).
+
+### RKE
+
+Minimum CPU and memory requirements for each individual node in the [Kubernetes cluster Rancher is installed in](install-upgrade-on-a-kubernetes-cluster.md) are listed in the table below.
+
+Please note that a highly available setup with at least 3 nodes is required for all production usages.
+
+| Managed Infrastructure Size | Clusters  | Nodes      | vCPUs | RAM   |
+|-----------------------------|-----------|------------|-------|-------|
+| Small                       | Up to 150 | Up to 1500 | 4     | 16 GB |
+| Medium                      | Up to 300 | Up to 3000 | 8     | 32 GB |
+| Large (*)                   | Up to 500 | Up to 5000 | 16    | 64 GB |
+
+(*): Large deployments require [additional tuning and following of best practices](../reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md).
```

Comment:
```suggestion
(*): Large deployments should [follow best practices](../reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md) for running Rancher at scale.
```

Reply:
https://github.com/rancher/rancher-docs/pull/896/files#r1342999088

Reply:
Addressed as of dddf2b28c45148a1246891066e418577238e725d